hal/src/nRF52840/lwip/lwippppopts.h

```diff
@@ -127,7 +131,7 @@
 /**
  * PPP_IPV6_SUPPORT==1: Enable PPP IPv6 support
  */
-#define PPP_IPV6_SUPPORT                (LWIP_IPV6)
+#define PPP_IPV6_SUPPORT                0
```

Comment:
General question - Looks like we need to turn on IPV6 packet support manually if it is needed. In general, how do we figure out that we need this logging?

Reply:
We don't support IPv6 anyway at the moment on any interface other than Mesh. This completely disables PPP IPv6 support, not just the logging. We can always flip the switch back any time needed.